src/EA.Weee.RequestHandlers/AatfReturn/GetReturnStatusHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace EA.Weee.RequestHandlers.AatfReturn
+{
+    using System;
+    using System.Threading.Tasks;
+    using Prsd.Core.Mapper;
+    using Prsd.Core.Mediator;
+    using Requests.AatfReturn;
+    using Security;
+    using ReturnStatus = Core.AatfReturn.ReturnStatus;
+
+    internal class GetReturnStatusHandler : IRequestHandler<GetReturnStatus, ReturnStatus>
+    {
+        private readonly IWeeeAuthorization authorization;
+        private readonly IReturnDataAccess returnDataAccess;
+        private readonly IMapper mapper;
+
+        public GetReturnStatusHandler(IWeeeAuthorization authorization, 
+            IReturnDataAccess returnDataAccess, 
+            IMapper mapper)
+        {
+            this.authorization = authorization;
+            this.returnDataAccess = returnDataAccess;
+            this.mapper = mapper;
+        }
+
+        public async Task<ReturnStatus> HandleAsync(GetReturnStatus message)
+        {
+            authorization.EnsureCanAccessExternalArea();
```

Comment:
Had a look in the submit return handler tests and couldn't see tests to cover this handler e.g. ensure user access external area, making sure the error is thrown when @return is null etc.

Reply:
Added unit tests